### ISSUE TITLE
NTR: AICA-ribotide high in erythrocytes (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8325]
New term request AICA-ribotide high in erythrocytes (elthat_130330192714): AICAR; RBCs, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal

AICA-riboTide rather than AICA-riboside or AICA ribonucleotide ?
Please provide more context.

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/608688 "AICA-riboside is the nucleoside corresponding to AICAR (AICA-ribotide, also termed ZMP)" and on the corresponding clinical synopsis https://www.omim.org/clinicalSynopsis/608688 "Elevated erythrocyte AICAR (ZMP)".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000752
- label: Elevated erythrocyte AICA-ribotide concentration
- parents: [Abnormal erythrocyte metabolite concentration (HP:0034737)](https://hpo.jax.org/app/browse/term/HP:0034737)
- pmid: PMID:15114530
- def: Concentration of 5-amino-4-imidazolecarboxamide (AICA-riboside) in red blood cells is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8325
- synonyms:
